src/scene/layer.js

```diff
@@ -212,53 +113,241 @@ class Layer {
     constructor(options = {}) {
 
         if (options.id !== undefined) {
+            /**
+             * A unique ID of the layer. Layer IDs are stored inside {@link ModelComponent#layers},
```

Comment:
perhaps add RenderComponent#layers here too to make it complete.

Reply:
Fixed.